### PR TITLE
Add missing combat snapshots

### PR DESCRIPTION
**Pull request by jackassmc**
This PR adds "1.16_combat-1", "1.16_combat-2" and "1.16_combat-4".

### CODE REVIEW

**Comment by peterix** [comment]
They are already there.

**Comment by peterix** [comment]
AFAIK

**Comment by peterix** [comment]
AH, these are some other ones... I guess I actually missed a few

File: mojang/versions/1_16_combat-1.json [hunk 1]
```diff
@@ -0,0 +1,1250 @@
...
+        "url": "https://launcher.mojang.com/v1/objects/ef4f57b922df243d0cef096efe808c72db042149/client-1.12.xml"
+      },
+      "type": "log4j2-xml"
+    }
+  },
+  "mainClass": "net.minecraft.client.main.Main",
+  "minimumLauncherVersion": 21,
+  "releaseTime": "2020-08-12T14:07:25+00:00",
+  "time": "2020-08-12T14:07:25+00:00",
+  "type": "pending"
```
Comment (peterix) [inline mojang/versions/1_16_combat-1.json]: use "experiment"

File: mojang/versions/1_16_combat-4.json [hunk 2]
```diff
@@ -0,0 +1,1250 @@
...
+        "url": "https://launcher.mojang.com/v1/objects/ef4f57b922df243d0cef096efe808c72db042149/client-1.12.xml"
+      },
+      "type": "log4j2-xml"
+    }
+  },
+  "mainClass": "net.minecraft.client.main.Main",
+  "minimumLauncherVersion": 21,
+  "releaseTime": "2020-08-19T11:14:58+00:00",
+  "time": "2020-08-19T11:14:58+00:00",
+  "type": "pending"
```
Comment (peterix) [inline mojang/versions/1_16_combat-4.json]: use "experiment"

File: mojang/versions/1_16_combat-2.json [hunk 3]
```diff
@@ -0,0 +1,1250 @@
...
+        "url": "https://launcher.mojang.com/v1/objects/ef4f57b922df243d0cef096efe808c72db042149/client-1.12.xml"
+      },
+      "type": "log4j2-xml"
+    }
+  },
+  "mainClass": "net.minecraft.client.main.Main",
+  "minimumLauncherVersion": 21,
+  "releaseTime": "2020-08-13T11:56:30+00:00",
+  "time": "2020-08-13T11:56:30+00:00",
+  "type": "pending"
```
Comment (peterix) [inline mojang/versions/1_16_combat-2.json]: use "experiment"

**Comment by jackassmc** [comment]
Yeah, these are the missing "non-final" 1.16 combat snapshots. The final ones are already in here.

- 1.16_combat-0 = Combat Test 6
- 1.16_combat-1 = Combat Test 7 (new)
- 1.16_combat-2 = Combat Test 7b (new)
- 1.16_combat-3 = Combat Test 7c
- 1.16_combat-4 = Combat Test 8 (new)
- 1.16_combat-5 = Combat Test 8b
- 1.16_combat-6 = Combat Test 8c

**Review by peterix — CHANGES_REQUESTED** [review]
Please change type to `experiment`

**Comment by jackassmc** [comment]
> Please change type to `experiment`

Done.

**Pull request merged**